frontend/src/metabase/dashboard/dashboard.js

```diff
@@ -663,7 +666,6 @@ function expandInlineCard(card) {
 export const fetchDashboard = createThunkAction(FETCH_DASHBOARD, function(
   dashId,
   queryParams,
-  enableDefaultParameters = true,
```

Comment:
There are no `fetchDashboard` calls where this third arg is actually set, so I'm removing it.